src/lib/get-integration-badges.tsx

```diff
@@ -1,95 +1,49 @@
-/**
- * Copyright (c) HashiCorp, Inc.
- * SPDX-License-Identifier: MPL-2.0
- */
-
 import { IconArchive16 } from '@hashicorp/flight-icons/svg-react/archive-16'
 import { IconEnterprise16 } from '@hashicorp/flight-icons/svg-react/enterprise-16'
 import { IconHandshake16 } from '@hashicorp/flight-icons/svg-react/handshake-16'
 import { IconHashicorp16 } from '@hashicorp/flight-icons/svg-react/hashicorp-16'
 import { IconRocket16 } from '@hashicorp/flight-icons/svg-react/rocket-16'
 import { IconUsers16 } from '@hashicorp/flight-icons/svg-react/users-16'
 import { IconWrench16 } from '@hashicorp/flight-icons/svg-react/wrench-16'
-import classNames from 'classnames'
-import Badge, { BadgeProps } from 'components/badge'
-import Tooltip from 'components/tooltip'
-import {
-	Flag,
-	Integration,
-	Tier,
-} from 'lib/integrations-api-client/integration'
-import s from './style.module.css'
-
-export interface Tag {
-	name: string
-	icon?: React.ReactElement
-	description: string
-}
-
-interface TagListProps {
-	tags: Array<Tag>
-	className?: string
-	size?: BadgeProps['size']
-	type?: BadgeProps['type']
-}
-
-export default function TagList({
-	tags,
-	className,
-	size = 'small',
-	type = 'filled',
-}: TagListProps) {
-	return (
-		<ul className={classNames(s.tagList, className)}>
-			{tags.map((tag: Tag) => {
-				return (
-					<li key={tag.name} className={s.tag}>
-						<Tooltip label={tag.description}>
-							<Badge icon={tag.icon} text={tag.name} type={type} size={size} />
-						</Tooltip>
-					</li>
-				)
-			})}
-		</ul>
-	)
-}
+import { Badge } from 'components/badge-list'
+import { Flag, Integration, Tier } from './integrations-api-client/integration'
 
-export function GetIntegrationTags(
+// Converts an Integration object into an array of Badges to be used by our badge-list
+export function GetIntegrationBadges(
```

Comment:
⛏️ nitpick, I think this should be camelCase 
```suggestion
export function getIntegrationBadges(
```

Reply:
👍 you caught my Go habit of capitalizing thing that are exported

Reply:
(I will do this but need to adjust the imports too, so won't just accept the comment)